### PR TITLE
Opacity utility

### LINKED ISSUE #31592: [Utilities][Opacity] Add opacity classes
Easily change the opacity of elements with the opacity utilities.

This can be very useful in some context, for example : 

```html
<section class="module-list {% if not hook['active'] %} opacity-50 {% endif %}"> 
```
Instead of : 

```html
<section class="module-list" {% if not hook['active'] %} style="opacity:50%" {% endif %}> 
```

I can provide a PR if you are interested in this feature.

 - opacity-10     => opacity:10%
 - opacity-20     => opacity:20%
 - opacity-30     => opacity:30%
 - opacity-40     => opacity:40%
 - opacity-50     => opacity:50%
 - opacity-60     => opacity:60%
 - opacity-70     => opacity:70%
 - opacity-80     => opacity:80%
 - opacity-90     => opacity:90%
 - opacity-100   => opacity:100%

### PR DESCRIPTION
**Pull request by MauricioHernanCabrera**
Fixes #31592

### CODE REVIEW

**Review by ffoodd — CHANGES_REQUESTED** [review]
Do not commit `dist` files, please.

**Comment by MauricioHernanCabrera** [comment]
> Do not commit `dist` files, please.

okass. I'm going to do a rebase

**Comment by MauricioHernanCabrera** [comment]
> Do not commit `dist` files, please.

done!

**Review by PululuK — APPROVED** [review]

File: site/content/docs/5.0/utilities/opacity.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+layout: docs
+title: Opacity
+description: Use opacity utilities to quickly add transparency to the background of an element.
+group: utilities
+toc: true
+---
+
+## Opacity
+
+Opacity classes are available, though they are not responsive.
```
Comment (PululuK) [inline site/content/docs/5.0/utilities/opacity.md:11]: Opacity responsive. What does that mean 🤔  ?

Comment (MauricioHernanCabrera) [reply site/content/docs/5.0/utilities/opacity.md:11]: means it has no opacity break point, opacity-sm-10, opacity-md-20, etc

**Comment by MauricioHernanCabrera** [comment]
> The utility API documentation page uses opacity as an example, this should be adjusted if this lands. https://deploy-preview-31602--twbs-bootstrap.netlify.app/docs/5.0/utilities/api/
> 
> I'm also unsure if we really need such a large set of values.

I can do a list variable with values by default 0, 0.25, 0.50, 0.75

**Comment by mdo** [comment]
Putting this on hold for now given not much interest yet. Also agreed with @MartijnCuppens on too many values anyway.

**Comment by mdo** [comment]
Closing and passing on this for now.

**Pull request closed without merging**